src/main/java/org/folio/invoices/utils/ErrorCodes.java

```diff
@@ -5,6 +5,7 @@
 public enum ErrorCodes {
 
   GENERIC_ERROR_CODE("genericError", "Generic error"),
+  INCOMPATIBLE_INVOICE_FIELDS_ON_STATUS_TRANSITION("incompatibleInvoiceFields", "Invoice with statuses 'Open' or 'Reviewed' shouldn't contain fields 'approvedBy' or 'approvedDate'"),
```

Comment:
The error code is generic but description is not.

Reply:
updated description